src/components/uyghur/boycott/index.jsx

```diff
@@ -0,0 +1,91 @@
+import React, { Component } from 'react';
+
+// semantic-ui
+import {
+    Container, Icon, Label, Menu, Table 
+  } from 'semantic-ui-react';
+
+class UyghurBoycott  extends Component {
+  render() {
+    const color = site.variables.get(pageBackground);
+    return (
+        <Container style={{ fontSize: '20px'}}>
+        <p> 
+            Link to ASPI report: <a href="https://www.aspi.org.au/report/uyghurs-sale">https://www.aspi.org.au/report/uyghurs-sale</a>
+        </p>
+        <p class= "quote"> <i>  
+            “The Chinese government has facilitated the mass transfer of Uyghur and other ethnic minority citizens from the far west region of Xinjiang to factories across the country. Under conditions that strongly suggest forced labour, Uyghurs are working in factories that are in the supply chains of at least 83 well-known global brands in the technology, clothing and automotive sectors, including Apple, BMW, Gap, Huawei, Nike, Samsung, Sony and Volkswagen.”
+        </i>
+        </p>
+        <br></br>
+        <p> <b>
+            List of orgs to boycott:</b>
+        </p>
+        <Table celled color= {color} key ="#F9F9F9" inverted>
```

Comment:
```suggestion
        <Table celled color="primary" key ="#F9F9F9" inverted>
```

I suggest just using the default primary or secondary colors that I already have set up with semantic. If there is a specific color that you think would look much better, I can add it to the semantic build. But I would still highly recommend just using those defaults.

Reply:
yes, I was looking for what those are called! what is the cream-yellow bg color called?

Reply:
Our color palette can be found here! Sorry for not showing you that yet, but there is a folder called Website Layout which also has other things related to our site like our logos for instance

https://docs.google.com/document/d/1B86r3P9PZ7VHmQqorNP4MCg9G2srR1dl0ljghzOfqyk/edit?usp=sharing